### PR TITLE
LON-856

### PR DESCRIPTION
**Pull request by dfernandezramos**

### CODE REVIEW

File: LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs [hunk 1]
```diff
@@ -283,5 +285,15 @@ void EmitRenderPlaylist (List<TimelineEvent> plays)
 				}
 			);
 		}
+
+		bool checkLineupEvent ()
```
Comment (vguzmanp) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: methods should be in `PascalCase`, not `camelCase`.
This should be named `IsLineupEvent`

File: LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs [hunk 2]
```diff
@@ -283,5 +285,15 @@ void EmitRenderPlaylist (List<TimelineEvent> plays)
...
+		{
+			foreach (var p in this.plays) {
+				if (p is LineupEvent) {
```
Comment (vguzmanp) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: Better to use LINQ: `if(plays.Any(p => p is LineupEvent))`

Comment (vguzmanp) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: Are substitution events considered? They shouldn't be added either

**Comment by dfernandezramos** [comment]
Ready to review again

File: LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs [hunk 3]
```diff
@@ -283,5 +285,13 @@ void EmitRenderPlaylist (List<TimelineEvent> plays)
 				}
 			);
 		}
+
+		bool IsLineupEvent ()
+		{
+			if (plays.Any (p => p is LineupEvent || p is SubstitutionEvent)) {
+				return true;
+			}
+			return false;
```
Comment (vguzmanp) [inline LongoMatch.GUI/Gui/Menu/SportsPlaysMenu.cs]: `return plays.Any...`

**Comment by vguzmanp** [comment]
Squash the first `update submodule` and :+1:

**Pull request merged**